Assignment2/bgDani-DanielBerrocal.ttl

```diff
@@ -0,0 +1,10 @@
+@base <http://www.oeg-upm.net/Organization/>
+@prefix pre: <http:/www.oeg-upm.net/Organization#/>
+
+:Class01            pre:includes:Sensor029;
+                    pre:includes:Computer101.
+:Sensor029          pre:hasMeasurement:Measurement8401.
+:Computer101        pre:hasOwner:User10A.
+:Measurement8401    pre:hasTemperature:"29"^^xs:integer;
```

Comment:
You have to declare the prefix. And it is normally xsd
At the same time, there is a syntax error. Have you tried with a validator?